_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+
+### Argo Workflows Upgrade
+
+Kubeflow Pipelines’s Argo Workflows backend is [upgraded to 3.4.16](https://github.com/kubeflow/pipelines/issues/10469). This upgrade moves the supported version closer to the latest upstream version and brings lots of CVE resolutions. The previous minor version was no longer being patched by the Argo community, so lots of security issues had accumulated over time.
+
+### v1 Feature Parity
+
+We made a huge progress towards KFPv1 feature parity by adding more Kubernetes  resources to the Pipelines code with the new kfp-kubernetes 1.2.0 Python package.
+
+## Katib
+
+Kubeflow 1.9 ships with Katib 0.17, which brings [official support](https://github.com/kubeflow/katib/pull/2315) for ARM64, getting us one step closer to full ARM64 coverage.
+
+For Data Scientists who submit training jobs with the Python SDK, you can now set the [algorithm settings](https://github.com/kubeflow/katib/pull/2227) and [environment variables](https://github.com/kubeflow/katib/pull/2235) from the tune method. Previously, you had to rely directly on Kubernetes CRD submission for these. You can also take advantage of the latest features from TensorFlow 2.16 and PyTorch 2.2. The team also worked to resolve [environmental conflicts](https://github.com/kubeflow/katib/issues/2346) that prevented the Katib Python SDK to be installed alongside the Kubeflow Python SDK.
+
+There are tons of additional improvements and bug fixes. Check out the full changelog [here](https://github.com/kubeflow/katib/blob/master/CHANGELOG.md).
+
+## Central Dashboard
+
+This release bring several improvements to the Kubeflow Central Dashboard, including:
+
+- Styling improvements to the sidebar, including [grouping](https://github.com/kubeflow/kubeflow/pull/7583) all Kubeflow Pipelines links to reduce clutter
+- Significant [improvements](https://github.com/kubeflow/kubeflow/pull/7582) to the “manage contributors” page, including the ability to manage contributors for all profiles that you are the owner of, and see which profiles you have access to, even when you are not the owner
+- Allow external services to [parse](https://github.com/kubeflow/kubeflow/pull/7138) the current profile (namespace) by sending the namespace selector value to non-iframed applications
+- Significant [updates](https://github.com/kubeflow/kubeflow/pull/7578) to dependencies to reduce CVEs
+
+![Kubeflow notebook images](../images/2024-07-22-kubeflow-1.9-release/dashboard.png)
+
+## Notebooks
+
+With this release, we provide [significant updates](https://github.com/kubeflow/kubeflow/pull/7590) to all example notebook images including PyTorch 2.3.0, Tensorflow 2.15.1 and many other library updates. While you can continue to use the old images, we recommend updating to use the greatest and latest ML libraries.
```

Comment:
maybe a line to intro - Kubeflow Notebooks enable users to quickly spin up configurable model development environments  i.e. Jupyter, VSCode, R Studio notebooks with Tensorflow and PyTorch images and configurable infrastructure CPU/GPU, memory and storage resources.

Reply:
Josh, I'll mark this as resolved since I did not hear back. Feel free to re-open if you strongly believe we should add more intros.